ngx-tools/src/has-required-control/has-required-control.ts

```diff
@@ -16,7 +16,7 @@ import {
  * @param formItem - The control or form group to check
  * @return If a required control is found
  */
-export function hasRequiredControl(formItem: AbstractControl): boolean {
+export function hasRequiredControl(formItem: AbstractControl): formItem is AbstractControl {
```

Comment:
Let's actually remove this one. I had forgotten until I saw this, that we have another type coercion check for abstract control here:

https://github.com/GetTerminus/terminus-ui/blob/38137b20c05a1e3ab170a97aeca311193379890a/terminus-ui/src/utilities/type-coercion/is-abstract-control.ts

I'll open up a new ticket to move these three checks into the ngx-tools library. (they are only used by the library so it should not be a breaking change)